geomesa-tools/bin/common-functions.sh

```diff
@@ -302,6 +302,7 @@ function registerAutocomplete() {
 
 function geomesaScalaConsole() {
   classpath=${1}
+  OPTS=${@}
```

Comment:
will this also pick up the classpath arg?

Reply:
Yeah, my bad, it's fixed.

Reply:
It's not showing up though...